nemo/job.sh

```diff
@@ -8,6 +8,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Usage function to display help
+usage() {
+    echo "Usage: $0 --nnodes <NNODES> --config_path <CONFIG_PATH> --config_name <CONFIG_NAME> [other options...]"
+    echo
+    echo "Mandatory options:"
+    echo "  --nnodes <NNODES>                     Number of nodes (also sets +trainer.num_nodes)."
+    echo "  --config_path <CONFIG_PATH>           Path to config."
```

Comment:
nice! why not move job.sh one level up and only have ADDITIONA_ARGS file sit in llama2-7b and llama3-70b respectively?

Reply:
`ADDITIONA_ARGS` is to overwrite the configs on fly if you decide to do so - but it can also be empty and just modify the configs files directly.